README.md

```diff
@@ -37,6 +37,16 @@ You should set these up before attempting to install the repo.
 You will now have a Next.js development server running at http://localhost:3000, which will refresh with changes to your
 local environment.
 
+### Environment Flags
+
+The APP_ENV flag can be used to designate which .env file you want to use. Ensure you're env files are in ./env, and then run your build or dev command with a leading APP_ENV=local.
```

Comment:
I see you moved `.env.example` but not `.env.test`, just curious if `.env.test` gets picked up when it lives in this folder? We preface a bunch of package.json's `test:` scripts with `NODE_ENV=test`